submissions/Shrai-dev/html-css-popup/index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/main.css">
+    <link rel="shortcut icon" type="image/png" href="popup-icons/favicon.png"/>
```

Comment:
The single tags should be without slashes "/" (according to HTML5 specification)